jenkins-builds/6.0.UP04-17.08-mc

```diff
@@ -35,6 +35,7 @@
  ncview-2.1.7-CrayGNU-17.08.eb                      --set-default-module
  netcdf-python-1.2.9-CrayGNU-17.08-python2.eb
  netcdf-python-1.2.9-CrayGNU-17.08-python3.eb       --set-default-module
+ otfprofile-1.20.eb                                 --set-default-module
```

Comment:
Why this is visible to everyone? Was this a request from (a group of) users? 

If not, this should go with `--hidden`

Reply:
It is not a dependency, it is a tool that should be part of scorep, as it will help users to quickly check there traces before launching Vampir. I will add it to the recipe of scorep next time we build it but for now it should be visible.

Reply:
Our policy is that only 'supported software' should be visible. 

If you want it visible then please add it to the documentation on the user portal.